### PR TITLE
parse excel file and populate Intrebare & Optiune tables

### PR DESCRIPTION
**Pull request by LByron**
Excel file:
https://docs.google.com/spreadsheets/d/1kAhpqSu0bhEfxH0l0nWTOX0AlPcK_VXQbuDsj35WzL8/edit#gid=0

### CODE REVIEW

File: api/MonitorizareVot.Ong/src/MonitorizareVot.Domain.Ong/Models/USROngContextExtensions.cs [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+
+
+namespace MonitorizareVot.Domain.Ong
+{
+    public static class USROngContextExtensions
+    {
+
+        private static List<Sectiune> _sectiuni = new List<Sectiune>
+        {
+                new Sectiune { IdSectiune = 1, CodSectiune = "A", Descriere = "Deschidere" }
```
Comment (LByron) [inline api/MonitorizareVot.Ong/src/MonitorizareVot.Domain.Ong/Models/USROngContextExtensions.cs]: Care sunt sectiunile?

File: api/MonitorizareVot.Ong/src/MonitorizareVot.Domain.Ong/Models/USROngContextExtensions.cs [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+        }
+
+        private static List<Intrebare> GetIntrebariObservator(List<IntrebareExcel> intrebariExcel, List<Optiune> optiuni)
+        {
+            var intrebari = intrebariExcel.Select((x, index) => new Intrebare
+            {
+                IdIntrebare = index + 1,
+                CodIntrebare = x.CodIntrebare,
+                CodFormular = x.CodFormular.Split('-')[0],
+                IdSectiune = 1, // IMPORTANT: ce sectiune?
```
Comment (LByron) [inline api/MonitorizareVot.Ong/src/MonitorizareVot.Domain.Ong/Models/USROngContextExtensions.cs]: le voi adauga imediat ce aflu care sunt

File: api/MonitorizareVot.Ong/src/MonitorizareVot.Domain.Ong/ExcelParser.cs [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+                                    CodFormular = reader.GetString(1)?.Trim(),
+                                    CodIntrebare = reader.GetString(2)?.Trim(),
+                                    TextIntrebare = reader.GetString(3)?.Trim(),
+                                    IdTipIntrebare = reader.GetString(4)?.Trim(),
+                                    Optiuni = new List<OptiuneExcel>()
+                                };
+
+                             
+                          
+                                // Caz special pentru intrebari de tip number & text
```
Comment (LByron) [inline api/MonitorizareVot.Ong/src/MonitorizareVot.Domain.Ong/ExcelParser.cs]: conventia 1

File: api/MonitorizareVot.Ong/src/MonitorizareVot.Domain.Ong/ExcelParser.cs [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+                                } else {
+
+                                    var indexOptiune = 5;
+
+                                    while (indexOptiune < reader.FieldCount)
+                                    {
+                                        var optiune = reader.GetType() != typeof(String) ? reader.GetValue(indexOptiune)?.ToString() : reader.GetString(indexOptiune);
+                                        if (optiune != null && optiune != string.Empty)
+                                        {
+                                            //IMPORTANT: conventie la nivelul excelului, marcam optiune flagged appenduind la optiune "-flag"
```
Comment (LByron) [inline api/MonitorizareVot.Ong/src/MonitorizareVot.Domain.Ong/ExcelParser.cs]: a doua conventie

File: api/MonitorizareVot.Ong/src/MonitorizareVot.Domain.Ong/ExcelParser.cs (hunk 3 above)
Comment (bvizureanu) [reply api/MonitorizareVot.Ong/src/MonitorizareVot.Domain.Ong/ExcelParser.cs]: nu prea inteleg "tip number sau text" nu avem aceste tipuri, nu evidentiem nicaieri ce tip de data acceptam la freetext (e pe roadmap dar nu avem inca). Tipul intrebarii se refera la single answer, multiple answer, single answer with freetext, multiple answer with freetext vezi [aici](https://github.com/code4romania/monitorizare-vot-ong/blob/master/api/MonitorizareVot.Ong/src/MonitorizareVot.Domain.Ong/ValueObjects/TipIntrebareEnum.cs)

File: api/MonitorizareVot.Ong/src/MonitorizareVot.Domain.Ong/Models/USROngContextExtensions.cs [hunk 5]
```diff
@@ -0,0 +1,231 @@
...
+
+
+
+        private static void SeedJudete(this OngContext context)
+        {
+            if (context.Judet.Any())
+                return;
+
+            context.Judet.AddRange(
+                new Judet { IdJudet = 0, CodJudet = "AB", Nume = "ALBA" },
```
Comment (bvizureanu) [inline api/MonitorizareVot.Ong/src/MonitorizareVot.Domain.Ong/Models/USROngContextExtensions.cs]: ⚠️ IdJudet trebuie sa inceapa de la 1. asta a ramas aici in cod intr-un mod bizar :)

**Comment by RaduCStefanescu** [comment]
We are working on a Forms feature in the new API in 
`https://github.com/code4romania/monitorizare-vot/tree/refactor/major-wip`
I am closing this PR.

**Pull request closed without merging**